README.rst

```diff
@@ -40,6 +40,41 @@ Single-node influxdb, SSL for http frontend:
               key_file: /etc/influxdb/ssl/key.pem
               cert_file: /etc/influxdb/ssl/cert.pem
 
+Single-node influxdb with an admin user (only one admin user is supported
+currently):
+
+.. code-block:: yaml
+
+    influxdb:
+      server:
+        enabled: true
+        http:
+          enabled: true
+          bind:
+            address: 0.0.0.0
+            port: 8086
+        admin:
+          enabled: true
+          bind:
+            address: 0.0.0.0
+            port: 8083
+          user:
+            name: root
+            password: secret
+
+Single-node influxdb with a new database and its non-admin user:
+
+.. code-block:: yaml
+
+    influxdb:
+      server:
+        databases:
```

Comment:
This model is not working because we need to be able to configure the name of the DB using some parameters. The other point that is not good is to add user in the declaration of the DB. I will rewrite the model as follow allowing to

1. create databases
1. create user and associate databases (created previously)

Here is the model:
```
Single-node influxdb with a new databases:

.. code-block:: yaml

    influxdb:
      server:
        databases:
          mydb1:
            name: mydb1
            enabled: true
          mydb2:
            name: mydb2
            enabled: true

Single-node influxdb with new users:

.. code-block:: yaml

    influxdb:
      server:
        users:
          newusr1:
            name: newusr1
            password: secret1
            databases:
            - mydb1
          newusr2:
            name: newusr2
            password: secret2
            databases:
            - mydb1
            - mydb2
```